app/swagger/swagger/requests/form1010_ezr_attachments.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Swagger
+  module Requests
+    class Form1010EzrAttachments
+      include Swagger::Blocks
+
+      swagger_path '/v0/form1010_ezr_attachments' do
+        operation :post do
+          extend Swagger::Responses::BadRequestError
+          extend Swagger::Responses::UnprocessableEntityError
+          extend Swagger::Responses::AuthenticationError
```

Comment:
These extend calls allow you to inject basic error codes and preset example responses into the swagger doc.